### PR TITLE
Adds Garfield and Lasagna

### PR DESCRIPTION
**Pull request by SweptWasTaken**
My last branch got corrupted by my own stupidity so I had to wipe the branch and thus the pr and start this one. 

I have removed lasagnium as it was stupid and a placeholder and you can now just shove lasagna in the autolathe now thanks to Zumorica being awesome. It's even got it's own animation. 

Anyway this is confined example of adding lathe recipes, new materials, and lathe animations so it was useful to me and should be useful to anyone in the future. The garfield suit will probably be a clown costume in the future.

Discord_2020-04-16_16-32-54

Pic unrelated

### CODE REVIEW

**Comment by SweptWasTaken** [comment]
This is ready for a merge

**Comment by PJB3005** [comment]
I will merge this on the condition that you do not see the garfield recipes in the autolathe unless you insert a lasagna first.

**Comment by Zumorica** [comment]
> I will merge this on the condition that you do not see the garfield recipes in the autolathe until you insert a lasagna.

That will require changing the lathe code. If you need help with that, Swept, feel free to mention me

**Comment by SweptWasTaken** [comment]
> I will merge this on the condition that you do not see the garfield recipes in the autolathe until you insert a lasagna.

Understood

**Comment by PKPenguin321** [comment]
Warning, you said the r***** word in your OP, github banbots are on the lookout for that

**Comment by SweptWasTaken** [comment]
> Warning, you said the r***** word in your OP, github banbots are on the lookout for that

Good catch

File: Resources/Prototypes/materials.yml [hunk 1]
```diff
@@ -17,3 +17,12 @@
   electricresistivity: 1.0e+13
   thermalconductivity: 0.9
   specificheat: 840
+
+- type: material
+  id: lasagnamaterial
+  name: Lasagna
+  icon: Objects/Materials/lasagna.png
+  density: 6969
```
Comment (Iamgoofball) [inline Resources/Prototypes/materials.yml:25]: strongest material known to man: old pasta that was left out on the kitchen counter for 3 days

**Comment by SweptWasTaken** [comment]
Not funny. Didn't laugh.

**Comment by Zumorica** [comment]
I'm so sad you didn't repurpose this. But go off I guess :pensive:

**Comment by SweptWasTaken** [comment]
> 
> 
> I'm so sad you didn't repurpose this. But go off I guess 😔

Soon

**Pull request closed without merging**